### DISCUSSION TITLE
Root Login in clients

### DISCUSSION THREAD

**Discussion by ikki0123** [ltsp/ltsp#780] [General] [answered]
Hello everyone, I'm having trouble logging in as root on the client, the password I use for root on the server is not working.
Is there a default password for root on clients? Or do I have to do some configuration in ltsp.conf?

**Comment by alkisg** [comment, marked as answer]
Hi, see the last paragraph of https://ltsp.org/man/ltsp.conf/

**Reply by ikki0123** [reply]
thank you!!!
